Tests/KotlinStandAloneTests/Resources/test.kt

```diff
@@ -0,0 +1,61 @@
+interface I: TypeLocalized, SourceCodeIdentifiable
+
+interface TypeLocalized {
+    val localized: String
```

Comment:
How do string localisations work in Android? Are they string-key based?

Reply:
Its static xml files. So realistically in our implementation we'd have to sort it ourselves